tests/test_filters.py

```diff
@@ -425,6 +425,43 @@ def test_filters_document_type(self, update):
         assert Filters.document.category("application/")(update)
         assert Filters.document.mime_type("application/x-sh")(update)
 
+    def test_filters_file_extension(self, update):
```

Comment:
please add a test that makes sure the dot before the extension is respected, i.e. `file.foojpg` should not pass `Filters.file_extension('.jpg')`

Reply:
I've already done it.